docs/development/writingzeppelininterpreter.md

```diff
@@ -138,15 +150,24 @@ You can only omit your interpreter group when your interpreter group is selected
 codes for myintp2
 ```
 
-
-
-
 ### Examples
 
-Check some interpreters shipped by default.
+Checkout some interpreters released with Zeppelin by default.
 
  - [spark](https://github.com/apache/incubator-zeppelin/tree/master/spark)
  - [markdown](https://github.com/apache/incubator-zeppelin/tree/master/markdown)
  - [shell](https://github.com/apache/incubator-zeppelin/tree/master/shell)
  - [hive](https://github.com/apache/incubator-zeppelin/tree/master/hive)
 
+### Contributing a new Interpreter to Zeppelin releases
+
+We welcome contribution to a new interpreter. Please follow these few steps:
+
+ - First, check out the general contribution guide [here](./howtocontributewebsite.html).
+ - Follow the steps in "Make your own Interpreter" section above.
+ - Add your interpreter as in the "Configure your interpreter" section above; also add it to the example template [zeppelin-site.xml.template](https://github.com/apache/incubator-zeppelin/blob/master/conf/zeppelin-site.xml.template).
+ - Add tests! They are run by Travis for all changes and it is important that they are self-contained.
+ - Include your interpreter as a module in [`pom.xml`](https://github.com/apache/incubator-zeppelin/blob/master/pom.xml).
+ - Add documentation on how to use your interpreter under `docs/interpreter/`. Follow the Markdown style as this  [example](https://github.com/apache/incubator-zeppelin/blob/master/docs/interpreter/elasticsearch.md). Make sure you list config settings and provide working examples on using your interpreter in code boxes in Markdown. Link to images as appropriate (images should go to `docs/assets/themes/zeppelin/img/docs-img/`). And add a link to your documentation in the navigation menu (`docs/_includes/themes/zeppelin/_navigation.html`).
```

Comment:
There is something strange about this - I checked it's not showing up with double space
![image](https://cloud.githubusercontent.com/assets/8969467/12774255/ec7d5eb6-c9f7-11e5-852b-42d225a8262a.png)